From 6b2d2e0a71d3ebbb608c79f9ff59302cbcc3569e Mon Sep 17 00:00:00 2001
From: Brad Geltz <brad.geltz@intel.com>
Date: Wed, 7 Oct 2020 11:24:14 -0700
Subject: [PATCH 2/2] Use build environment vars
#
#  Copyright (c) 2015 - 2024, Intel Corporation
#  SPDX-License-Identifier: BSD-3-Clause
#

- Updated AVX2 and SKX setup for IMPI_IOMP.

Signed-off-by: Brad Geltz <brad.geltz@intel.com>
---
 setup/Make.IMPI_IOMP_AVX2 | 2 +-
 setup/Make.IMPI_IOMP_SKX  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup/Make.IMPI_IOMP_AVX2 b/setup/Make.IMPI_IOMP_AVX2
index 38b0f73..bff2726 100644
--- a/setup/Make.IMPI_IOMP_AVX2
+++ b/setup/Make.IMPI_IOMP_AVX2
@@ -134,7 +134,7 @@ HPCG_DEFS     = -DMPICH_IGNORE_CXX_SEEK $(HPCG_OPTS) $(HPCG_INCLUDES)
 # - Compilers / linkers - Optimization flags ---------------------------
 # ----------------------------------------------------------------------
 #
-CXX          = mpiicpc
+CXX          = $(MPICXX)
 CXXFLAGS     = -xCORE-AVX2 -qopenmp -std=c++11 $(HPCG_DEFS)
 ifeq (yes, $(DBG))
   CXXFLAGS  += -O0 -g -DHPCG_DEBUG
diff --git a/setup/Make.IMPI_IOMP_SKX b/setup/Make.IMPI_IOMP_SKX
index 9325d71..edc99cd 100644
--- a/setup/Make.IMPI_IOMP_SKX
+++ b/setup/Make.IMPI_IOMP_SKX
@@ -134,7 +134,7 @@ HPCG_DEFS     = -DMPICH_IGNORE_CXX_SEEK $(HPCG_OPTS) $(HPCG_INCLUDES)
 # - Compilers / linkers - Optimization flags ---------------------------
 # ----------------------------------------------------------------------
 #
-CXX          = mpiicpc
+CXX          = $(MPICXX)
 CXXFLAGS     = -xCORE-AVX512 -qopt-zmm-usage=high -qopenmp -std=c++11 $(HPCG_DEFS)
 ifeq (yes, $(DBG))
   CXXFLAGS  += -O0 -g -DHPCG_DEBUG
--
2.23.0

